pytorch/inference/docker/1.9/py3/sdk1.16.0/Dockerfile.neuron

```diff
@@ -0,0 +1,160 @@
+    mv /etc/ssl/certs/java/cacerts.jks /etc/ssl/certs/java/cacerts; \
+    /var/lib/dpkg/info/ca-certificates-java.postinst configure;
+
+RUN curl -L -o ~/miniconda.sh https://repo.continuum.io/miniconda/Miniconda3-latest-Linux-x86_64.sh \
+ && chmod +x ~/miniconda.sh \
+ && ~/miniconda.sh -b -p /opt/conda \
+ && rm ~/miniconda.sh \
+ && /opt/conda/bin/conda update conda \
+ && /opt/conda/bin/conda install -c conda-forge \
+    python=$PYTHON_VERSION \
+ && /opt/conda/bin/conda install -y \
+    # conda 4.9.2 requires ruamel_yaml to be installed. Currently pinned at latest.
+    ruamel_yaml==0.15.100 \
+    cython==0.29.12 \
+    ipython==7.7.0 \
+    mkl-include==2019.4 \
+    mkl==2019.4 \
+    numpy==1.19.1 \
+    scipy==1.3.0 \
+    typing==3.6.4 \
+    parso==0.8.0 \
+ && /opt/conda/bin/conda clean -ya
+
+RUN conda install -c \
+    conda-forge \
+    opencv==4.0.1 \
+ && conda install -y \
+    scikit-learn==0.21.2 \
+    pandas==0.25.0 \
+    h5py==2.9.0 \
+    requests==2.22.0 \
+ && conda clean -ya \
+ && pip install --upgrade pip --trusted-host pypi.org --trusted-host files.pythonhosted.org \
+ && ln -s /opt/conda/bin/pip /usr/local/bin/pip3 \
+ && pip install packaging==20.4 \
+    enum-compat==0.0.3 \
+    # pyOpenSSL requires cryptography>=2.3, but all versions <3.3 have vulnerabilities
+    "cryptography>=3.3.2"
+
+RUN pip install --no-cache-dir -U \
+    numpy==1.17.4 \
+    scipy==1.2.2 \
+    scikit-learn==0.20.3 \
+    pandas==0.24.2 \
+    h5py==2.10.0 \
+    # install PyYAML>=5.4 to avoid conflict with latest awscli
+    "pyYAML>=5.4,<5.5" \
+    cython==0.29.12 \
+    falcon==2.0.0 \
+    gunicorn==20.0.4 \
+    gevent==1.4.0 \
+    requests==2.22.0 \
+    grpcio==1.34.1 \
+    protobuf==3.16.0 \
+    gast==0.2.2 \
+    "pillow>=8.3" \
+    "awscli<2" \
+    boto3
+
+RUN pip install neuron-cc[tensorflow] --extra-index-url https://pip.repos.neuron.amazonaws.com \
+ && pip install "torch-neuron>=1.9.1,<1.9.2" --extra-index-url https://pip.repos.neuron.amazonaws.com \
+ && pip install torchserve==$TS_VERSION --no-deps \
+ && pip install --no-deps --no-cache-dir -U torchvision==0.10.1 \
+ # Install TF 1.15.5 to override neuron-cc[tensorflow]'s installation of tensorflow==1.15.0
+ && pip install -U tensorflow==1.15.5 \
+ && pip install torch-model-archiver==$TS_VERSION
+
+RUN useradd -m model-server \
+ && mkdir -p /home/model-server/tmp /opt/ml/model \
+ && chown -R model-server /home/model-server /opt/ml/model
+
+COPY neuron-entrypoint.py /usr/local/bin/dockerd-entrypoint.py
+COPY neuron-start.sh /usr/local/bin/neuron-start.sh
+COPY torchserve-neuron.sh /usr/local/bin/entrypoint.sh
+COPY config.properties /home/model-server
+
+RUN chmod +x /usr/local/bin/dockerd-entrypoint.py
+RUN chmod +x /usr/local/bin/neuron-start.sh
+RUN chmod +x /usr/local/bin/entrypoint.sh
```

Comment:
nit: These can be merged into a single `RUN` statement to reduce the number of layers in the image built from this dockerfile.